web/src/components/pages/RedirectPage.tsx

```diff
@@ -0,0 +1,11 @@
+const RedirectPage = () => {
+  window.close()
```

Comment:
this is called once per creation. as a result I can't install it twice, we need it onMount  
https://docs.solidjs.com/reference/lifecycle/on-mount